scripts/AbstractWidget.py

```diff
@@ -0,0 +1,322 @@
+import re
+import sys
+import os
+class eventHandlerInfo:
+    def __init__(self, java_src, identity, java_line_number,xml_line_number = None, xml_src = None, name=None):
+        self.xml_src = xml_src
```

Comment:
`NamedTuple` (https://pymotw.com/2/collections/namedtuple.html) can simplify this class as well.

Reply:
I guess I can make the change. But that involves changing a lot of code... Do you want to change it really bad?

Reply:
No need, just FYI